packages/cli/src/cmds/preview/preview.ts

```diff
@@ -608,13 +611,14 @@ export default class Preview extends YargsCommand {
       : undefined;
 
     const dotnetExecPath = await dotnetChecker.getDotnetExecPath();
-    const authStartTask = includeFrontend
-      ? this.prepareTask(
-          TaskDefinition.authStart(dotnetExecPath, commonUtils.getAuthServicePath(localEnv)),
-          constants.authStartStartMessage,
-          commonUtils.getAuthLocalEnv(localEnv)
-        )
-      : undefined;
+    const authStartTask =
+      includeFrontend && includeAuth
```

Comment:
`includeAuth` has two meanings:
1. Need AAD and auth functionality
2. Need SimpleAuth service.

I believe the goal is to remove 2) only, so better to directly use `includeSimpleAuth`.

Reply:
Updated. Thx!